serve/graph/transaction.resolvers.go

```diff
@@ -0,0 +1,22 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.45
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gnolang/tx-indexer/serve/graph/model"
+)
+
+// Response is the resolver for the response field.
+func (r *transactionResolver) Response(ctx context.Context, obj *model.Transaction) (*model.TransactionResponse, error) {
+	panic(fmt.Errorf("not implemented: Response - response"))
+}
+
+// Transaction returns TransactionResolver implementation.
+func (r *Resolver) Transaction() TransactionResolver { return &transactionResolver{r} }
+
+type transactionResolver struct{ *Resolver }
```

Comment:
This is not needed. Add a Response() method to Transaction Model

Reply:
Thank you. 
I didn't see any code changes with the addition of the merge commit.